landing/js/welcome.js

```diff
@@ -125,6 +125,31 @@
           }
         ],
         id: 'attaching'
+      },
+	  {
+        title: 'Highliting multiple Elements',
+        text: 'Your tour steps can highlight multiple elements (like this step).',
+        attachTo: {
+          element: '.hero-multiple-select',
+		  on: 'bottom',
```

Comment:
Lots of weird spacing issues in here. Could you please auto format this with prettier?